pages/common/wget.md

```diff
@@ -11,26 +11,26 @@
 
 `wget -O {{bar}} {{https://example.com/foo}}`
 
-- Download a single web page and all its resources (scripts, stylesheets, images, etc.):
+- Download a single web page and all its resources with 3-second intervals between requests (scripts, stylesheets, images, etc.):
 
-`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
-
-- Download a full website, with 3-second intervals between requests:
-
-`wget --mirror --page-requisites --convert-links --wait=3 {{https://example.com}}`
+`wget --page-requisites --convert-links --wait=3 {{https://example.com/somepage.html}}`
```

Comment:
I don't know how much sense this example makes. Who would want to wait 3 seconds between each request when dumping a web page and its resources?